src/containers/LoginForm.js

```diff
@@ -56,7 +56,7 @@ export class LoginForm extends Component {
     ) : (
       <section className='login-section'>
         <div className='form-container'>
-          <h1>Login Form</h1>
+          <h1 className='form-title'>Login</h1>
           <h1 className='error-styling'>{this.state.error}</h1>
           {!this.state.error && <h1 className='error-styling'>{this.props.errorMessage}</h1>}
```

Comment:
@Dunn-Austen @peeratmac   No rush on this - just making a note so I don't forget when we get back to it this weekend.  I'm adding a snapshot for the conditional rendering for the LoginForm and realized that I don't know what line 61 should be doing?   

Based on how its written now it says `if there is NOT an error, display the errorMessage as an h1`  but if there is not an error, that will just be an empty string.  

Something to revisit when we're back at it.  Opening an issue so its on our PM board.

Reply:
The way this works out is that Line 61's conditional statement returns true if the local state's error property is an empty string, but false if the string is populated with an error message:

- !(' ') returns true
- !('Please fill out all input fields') returns false 

The code is meant to only display the store's error message for the failed post/login attempt in the case that both input fields have been filled out (meaning an empty string in the local state). If the user only populates the email field but leaves the password input empty (or vice versa), instead of receiving both the input validation error from the local state as well as the failed post/login attempt error from the store - he or she would only receive the message prompt to fill out both fields. 

The reasoning in keeping the error messages separately displayed involves a few things:
1) User clarity as to the specific failure (invalid email/password combo vs empty field)
2) Due to accompanying conditional statements attached to the form submit event, there is no attempted fetch request in the case where an input is left empty. 
3) Spacing considerations

Reply:
@Dunn-Austen Thank you for explaining.  I had a hard time wrapping my head around that line and your explanation was super clear and helpful!